sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -172,6 +216,39 @@ export interface MetricNamespace {
     type?: string;
 }
 
+// @public (undocumented)
+export interface MetricResultsResponseValuesItem {
+    endtime: string;
+    interval?: string;
+    namespace?: string;
+    resourceid?: string;
+    resourceregion?: string;
+    starttime: string;
+    value: BatchMetric[];
+}
+
+// @public
+export class MetricsBatchClient {
+    constructor(tokenCredential: TokenCredential, options?: MetricsBatchClientOptions);
+    batch(resourceids: string[], metricnamespace: string, metricnames: string[], options?: MetricsBatchOptionalParams): Promise<MetricResultsResponseValuesItem[]>;
```

Comment:
The method should be named `queryBatch` similar to the method we have in `LogsQueryClient`.

Reply:
Done. Will Update in the next commit